lib/cerebrum.rb

```diff
@@ -1,5 +1,47 @@
 require "cerebrum/version"
+require 'matrix'
 
 module Cerebrum
```

Comment:
look at the latest master, this was changed to `class`. the way it is right here makes the API call resolve to `Cerebrum::NeuralNetwork.new`, which would be fine but we're not really using the `Cerebrum` namespace for anything else.